make-util.js

```diff
@@ -331,6 +331,9 @@ exports.ensureTool = ensureTool;
 
 var installNode = function (nodeVersion) {
     switch (nodeVersion || '') {
+        case '16':
+            nodeVersion = 'v16.15.1';
```

Comment:
JFYI, we'll probably update it soon
https://github.com/microsoft/azure-pipelines-agent/issues/4075